### PR TITLE
Support return query result as a JSON string

### PR DESCRIPTION
**Pull request by Aiee**
Add `execute_json()` and tests

Depends on https://github.com/vesoft-inc/nebula/pull/2824

### CODE REVIEW

File: nebula2/gclient/net/Session.py [hunk 1]
```diff
@@ -39,24 +39,56 @@ def execute(self, stmt):
...
+        :param stmt: the ngql
+        :return: json string
+        """
+        if self._connection is None:
+            raise RuntimeError('The session has released')
+        try:
+            start_time = time.time()
+            resp_json = self._connection.execute_json(self._session_id, stmt)
+            end_time = time.time()
+            return resp_json
```
Comment (Nicole00) [inline nebula2/gclient/net/Session.py:75]: just return the bare result to user? The data returned by the server is bytes.

Comment (Aiee) [reply nebula2/gclient/net/Session.py:75]: After deserialization, the `resp_json` will be a readable JSON string. I'll add more tests when the server-side PR is merged.

Comment (Aiee) [reply nebula2/gclient/net/Session.py:116]: More cases have been added.

File: tests/test_data_from_server.py [hunk 2]
```diff
@@ -245,3 +247,48 @@ def test_path_type(self):
         assert resp.whole_latency() > 100
 
 
+class TestExecuteJson(TestBaseCase):
```
Comment (klay-ke) [inline tests/test_data_from_server.py:250]: add more TC for map/set/list?

**Review by klay-ke — DISMISSED** [review]
all others LGTM

File: tests/test_data_from_server.py (hunk 2 above)
Comment (Aiee) [reply tests/test_data_from_server.py:250]: Fixed.

Comment (klay-ke) [reply tests/test_data_from_server.py:250]: neat and clean code

File: nebula2/gclient/net/Session.py [hunk 3]
```diff
@@ -39,24 +39,97 @@ def execute(self, stmt):
...
+
+        :param stmt: the ngql
+        :return: JSON string
+        """
+        if self._connection is None:
+            raise RuntimeError('The session has released')
+        try:
+            start_time = time.time()
+            resp_json = self._connection.execute_json(self._session_id, stmt)
+            end_time = time.time()
```
Comment (yixinglu) [inline nebula2/gclient/net/Session.py:115]: what's the usage about the start_time and end_time?

File: nebula2/gclient/net/Session.py [hunk 4]
```diff
@@ -39,24 +39,97 @@ def execute(self, stmt):
...
                                      timezone_offset=self._timezone_offset)
             raise
         except Exception:
             raise
 
+    def execute_json(self, stmt):
+        """execute statement and return the result as a JSON string
+            Date and Datetime will be returned in UTC
+            JSON struct:
+              "results": [
```
Comment (HarrisChu) [inline nebula2/gclient/net/Session.py:67]: 1. how to determine if the query is successful? 
2. why the errors is in result content? how about using another key.
3. lack of error code?

File: nebula2/gclient/net/Session.py [hunk 5]
```diff
@@ -39,24 +39,97 @@ def execute(self, stmt):
...
+                    ],
+                    "nodeIndexMap" : {},
+                    "format" : "",
+                    "optimize_time_in_us" : 0,
+                  },
+                  "comment ": "",
+                  "errors" : "" // errorMsg
+                }
+              ]
+            	}]
```
Comment (HarrisChu) [inline nebula2/gclient/net/Session.py:105]: redundant line here?

File: nebula2/gclient/net/Session.py [hunk 6]
```diff
@@ -39,24 +39,97 @@ def execute(self, stmt):
...
+                        "isDoBranch" : false,
+                        "conditionNodeId" : -1,
+                      },
+                      "dependencies" : [] // vector of ints
+                      }
+                    ],
+                    "nodeIndexMap" : {},
+                    "format" : "",
+                    "optimize_time_in_us" : 0,
+                  },
```
Comment (Nicole00) [inline nebula2/gclient/net/Session.py:100]: Are all messages useful for users(such as `planDesc`)? if not, is it better to simply the json content returned to users?

File: nebula2/gclient/net/Session.py (hunk 4 above)
Comment (HarrisChu) [reply nebula2/gclient/net/Session.py:67]: in test case, error has two keys, errorCode and errorMsg.
should enhance the comment.

Comment (Aiee) [reply nebula2/gclient/net/Session.py:67]: 1. errorCode can be used to determine the status
2. This is to follow the neo4j JSON format
3. Good catch, errorCode is under the `errors`. I have added it back to the comment.

File: nebula2/gclient/net/Session.py (hunk 3 above)
Comment (Aiee) [reply nebula2/gclient/net/Session.py:115]: These are used in the `execute` interface. Deleted.

File: nebula2/gclient/net/Session.py (hunk 6 above)
Comment (Aiee) [reply nebula2/gclient/net/Session.py:100]: `planDesc` will only be filled when the user executes a query with `PROFILE/EXPLAIN`.
This JSON format is compatible with the neo4j format.

File: nebula2/gclient/net/Session.py (hunk 5 above)
Comment (Aiee) [reply nebula2/gclient/net/Session.py:105]: I have reformated the JSON. Thanks for pointing this out.

**Review by HarrisChu — APPROVED** [review]
LGTM

**Pull request merged**